servant/src/Servant/API/Empty.hs

```diff
@@ -0,0 +1,10 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+{-# OPTIONS_HADDOCK not-home    #-}
+module Servant.API.Empty(EmptyAPI(..)) where
+
+import           Data.Typeable    (Typeable)
+import           Prelude ()
+import           Prelude.Compat
+
+-- | An empty API: one which serves nothing.
```

Comment:
It's only "morally" the unit, though, right? It's still very much observable whether `EmptyAPI` is used or not.